readthedocs/projects/models.py

```diff
@@ -485,6 +485,14 @@ def single_version_symlink_path(self):
     #
 
     def venv_bin(self, version=LATEST, bin='python'):
```

Comment:
I'd prefer this, yes. I didn't want to have to refactor all of the calls, but searching again it looks like there aren't any calling python this way anyways.